### PR TITLE
Fix VPN-3424: Unable to assign [undefined] to QString

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
## Reference

    i.e Jira or Github issue URL

## Checklist
    
- [ ] My code follows the style guidelines for this project
- [ ] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [ ] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

### CODE REVIEW

**Review by MattLichtenstein — APPROVED** [review]

**Review by bakulf — APPROVED** [review]

File: nebula/ui/components/VPNLinkButton.qml [hunk 1]
```diff
@@ -11,7 +11,7 @@ import Mozilla.VPN 1.0
 VPNButtonBase {
     id: root
 
-    property var labelText
+    property var labelText: ""
```
Comment (bakulf) [inline nebula/ui/components/VPNLinkButton.qml:14]: Can we use `string` as type instead?

**Review by bakulf — APPROVED** [review]

File: nebula/ui/components/VPNLinkButton.qml (hunk 1 above)
Comment (MattLichtenstein) [reply nebula/ui/components/VPNLinkButton.qml:14]: this is actually a good point, I believe using the correct data type not only provides a slight performance boost, but also allows you to not need to specify an `""` to denote that the string is defined, but empty

Comment (bakulf) [reply nebula/ui/components/VPNLinkButton.qml:14]: Do you think it makes sense to audit the use of `var` and replace them with proper types? (not in this PR, not in this jira issue)

Comment (MattLichtenstein) [reply nebula/ui/components/VPNLinkButton.qml:14]: It doesn't seem to be documented anywhere just how much of a performance degradation this causes, but it's enough for [the syntax checker look for them (id: M311)](https://doc.qt.io/qtcreator/creator-checking-code-syntax.html). That and we won't run into the QML warning that this PR sets out to fix as much. I tend to only use `var` when none of the other basic types would suffice

Comment (lesleyjanenorton) [reply nebula/ui/components/VPNLinkButton.qml]: > allows you to not need to specify an `""` to denote that the string is defined, but empty

Nice. updated. +1 to audit.

**Pull request merged**